modules/core/src/v2/coins/algo.ts

```diff
@@ -453,7 +505,38 @@ export class Algo extends BaseCoin {
     return accountLib.Algo.algoUtils.decodeAlgoTxn(txn);
   }
 
-  getAddressFromPublicKey(Pubkey: Uint8Array): string {
-    return accountLib.Algo.algoUtils.publicKeyToAlgoAddress(Pubkey);
+  getAddressFromPublicKey(pubKey: Uint8Array): string {
+    return accountLib.Algo.algoUtils.publicKeyToAlgoAddress(pubKey);
+  }
+
+  /**
+   * Stellar and Algorand both use keys on the ed25519 curve, but use different encodings.
+   * As the HSM doesn't have explicit support to create Algorand addresses, we use the Stellar
+   * keys and re-encode them to the Algorand encoding.
+   *
+   * This method should only be used when creating Algorand custodial wallets reusing Stellar keys.
+   *
+   * @param {string} addressOrPubKey a Stellar pubkey or Algorand address
+   * @return {*}
+   */
+  private stellarAddressToAlgoAddress(addressOrPubKey: string): string {
```

Comment:
Not a blocker but will there be a needed use case for `algoAddressToStellarAddress` for sending data back through our service layer in the accepted Stellar format?